components/layout_2020/replaced.rs

```diff
@@ -63,6 +64,41 @@ impl ReplacedContent {
         None
     }
 
+    fn flow_relative_intrinsic_size(&self, style: &ComputedValues) -> Vec2<Option<Length>> {
+        let intrinsic_size = physical::Vec2 {
+            x: self.intrinsic_width,
+            y: self.intrinsic_height,
+        };
+        intrinsic_size.size_to_flow_relative(style.writing_mode())
+    }
+
+    fn inline_size_over_block_size_intrinsic_ratio(
+        &self,
+        style: &ComputedValues,
+    ) -> Option<CSSFloat> {
+        self.intrinsic_ratio.map(|width_over_height| {
```

Comment:
Sorry, but again, I find the code more readable when that is just named `ratio`.